docs/requests.md

```diff
@@ -116,6 +116,45 @@ In some cases such as long-polling, or streaming responses you might need to
 determine if the client has dropped the connection. You can determine this
 state with `disconnected = await request.is_disconnected()`.
 
+#### Request Files
+
+Request files are normally sent as multipart form data (`multipart/form-data`).
+
+When you call `await request.form()` you receive a `starlette.datastructures.FormData` which is an immutable
+multidict, containing both file uploads and text input.
+
+When one of the fields is a file upload, you will get a `starlette.datastructures.UploadFile`.
```

Comment:
We could shorten this line and the preceeding one, into a single para. Perhaps. *"containing both file uploads and text input. File upload items are represented as instances of `starlette.datastructures.UploadFile`."*?

Reply:
Sure thing.